lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -321,6 +323,9 @@ module InternalConstants
                  :default => nil,
                  :secret => true
 
+    # The URL of Stackdriver Logging API.
```

Comment:
This theoretically appears in some generated parameter description somewhere. Add something like "`Right now this only works with the gRPC path (use_grpc = true).`"?

Reply:
Added.